src/game/protinst.cc

```diff
@@ -406,21 +418,17 @@ int obj_examine_func(Object* critter, Object* target, void (*fn)(char* string))
 
                 snprintf(formattedText, sizeof(formattedText), v66.text, hpMessageListItem.text);
             } else {
-                // %s %s
-                v66.num = 521 + v12;
```

Comment:
I've found no correct way to use messages **521** - `%s %s.` and **519** - `%s %s `.
Seems like the first `%s` is for `He/She/It looks:` and the second `%s` is for examine text `hpMessageListItem`, but the messages file doesn't have separate messages for the first one - only with format parameter and a dot.
So, I think, without changing messages file it is not possible to correctly format the text taking into account if the critter is crippled or not.

The version of Fallout that is delivered through Xbox app has the same functionality as your fix.